dap_job_quality/pipeline/prodigy/README.md

```diff
@@ -0,0 +1,50 @@
+## Label job ads for dimensions of job quality
+
+### Generate data to label
+
+To generate data to label of size `ts` and save the data locally to an `.jsonl` file, run:
+
+```
+python dap_job_quality/pipeline/prodigy/make_labelled_data.py -ts 1000
+```
+
+If you would like to also save the data to s3, run:
+
+```
+python dap_job_quality/pipeline/prodigy/make_labelled_data.py -ts 1000 -s3 True
+```
+
+### Download BENEFITS model
+
+To download the NER model that extracts `BENEFITS`, run:
+
+```
+aws s3 cp s3://open-jobs-lake/escoe_extension/outputs/models/ner_model/20230808/ ./outputs/models/ner_model/20230808/ --recursive
+```
+
+### Run prodigy instance
+
+To install prodigy, run:
+
+```
+python -m pip install prodigy -f https://XXXX-XXXX-XXXX-XXXX@download.prodi.gy
+```
+
+To run the custom prodigy instance (in the `prodigy` directory), run:
+
+```
+prodigy benefits_classification job_quality_annotated \
+    ./labelled_data/20240117_ads_to_label_ts_1000_random_seed_42.jsonl \
+    -F custom_recipe.py
+```
```

Comment:
I found this a bit fidly, because I had to run everything else from the root directory - could we update this to do the same? 
```prodigy benefits_classification job_quality_annotated \
    ./dap_job_quality/pipeline/prodigy/labelled_data/20240118_ads_to_label_ts_1000_random_seed_42.jsonl \
    -F dap_job_quality/pipeline/prodigy/custom_recipe.py
```
```

Reply:
i know relative paths are such a drag - I had issues with running the recipe from the root directory :( so that's why i did it this way.